dev/benchmarks/platform_views_layout/lib/main_ad_banners.dart

```diff
@@ -0,0 +1,62 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+
+import 'package:flutter/material.dart';
+import 'package:google_mobile_ads/google_mobile_ads.dart';
+
+void main() {
+  runApp(
+    const PlatformViewApp()
+  );
+}
+
+class PlatformViewApp extends StatefulWidget {
+  const PlatformViewApp({
+    super.key,
+  });
+
+  @override
+  PlatformViewAppState createState() => PlatformViewAppState();
+}
+
+class PlatformViewAppState extends State<PlatformViewApp> {
+
+  AdWidget _getBannerWidget() {
+    final String bannerId = Platform.isAndroid
+        ? 'ca-app-pub-3940256099942544/6300978111'
+        : 'ca-app-pub-3940256099942544/2934735716';
```

Comment:
Can you explain this?

Reply:
These are test IDs provided by AdMob. Let me add a comment.

Reply:
This was resolved but I don't see a comment here yet?

Reply:
Odd. I did push the code before re-requesting review. Likely a race condition.